julia_src/ghpghx_inputs.jl

```diff
@@ -0,0 +1,251 @@
+# Using same struct type from REopt utils.jl
+# Warning: I've seen notes of Base.@kwdef not being officially supported (not documented)
+
+Base.@kwdef struct InputsStruct
+    ##### These are the exact /ghpghx POST names from the API #####
+    # Parameters
+    borehole_depth_ft::Float64
+    ghx_header_depth_ft::Float64
+    borehole_spacing_ft::Float64
+    borehole_diameter_inch::Float64
+    borehole_spacing_type::String  # "rectangular" or "hexagonal"
+    ghx_pipe_outer_diameter_inch::Float64
+    ghx_pipe_wall_thickness_inch::Float64
+    ghx_pipe_thermal_conductivity_btu_per_hr_ft_f::Float64
+    ghx_shank_space_inch::Float64
+    ground_thermal_conductivity_btu_per_hr_ft_f::Float64
+    ground_mass_density_lb_per_ft3::Float64
+    ground_specific_heat_btu_per_lb_f::Float64
+    grout_thermal_conductivity_btu_per_hr_ft_f::Float64
+    ghx_fluid_specific_heat_btu_per_lb_f::Float64
+    ghx_fluid_mass_density_lb_per_ft3::Float64
+    ghx_fluid_thermal_conductivity_btu_per_hr_ft_f::Float64
+    ghx_fluid_dynamic_viscosity_lbm_per_ft_hr::Float64
+    ghx_fluid_flow_rate_gpm_per_ton::Float64
+    ghx_pump_power_watt_per_gpm::Float64
+    ghx_pump_min_speed_fraction::Float64
+    ghx_pump_power_exponent::Float64
+    max_eft_allowable_f::Float64
+    min_eft_allowable_f::Float64
+    
+    # Array/Dict inputs
+    heating_thermal_load_mmbtu_per_hr::Array{Float64,1}
+    cooling_thermal_load_ton::Array{Float64,1}
+    ambient_temperature_f::Array{Float64,1}
+    cop_map_eft_heating_cooling::Array{Any,1}
+
+    # Model Settings
+    simulation_years::Int64  # Number of years for GHP-GHX model
+    solver_eft_tolerance_f::Float64  # Tolerance for the EFT error to accept a GHX sizing solution
+    solver_eft_tolerance::Float64  # Convert to degC
+    ghx_model::String  # "TESS" or "DST"
+    dst_ghx_timesteps_per_hour::Int64
+    tess_ghx_minimum_timesteps_per_hour::Int64
+    max_sizing_iterations::Int64
+    init_sizing_factor_ft_per_peak_ton::Float64
+    
+    ##### These are the variable names used in the GHPGHX, kept from TESS ######
+    # TODO eventually just use the API names above in GHPGHX to remove redundancy (BUT WOULD HAVE TO DEAL WITH UNITS CONVERSION STILL)  
```

Comment:
It would be really nice to have naming consistency.

Reply:
I agree! But, I think I'm going to categorize this as a "nice-to-have" and keep this as a TODO for now. I will tackle this next FY with the follow-on GTO funds when we do a little more development on this GHPGHX.jl module and I can align GHP with V2.